app/js/components/mock/service_edit_form.html

```diff
@@ -22,20 +33,28 @@ <h2>General</h2>
                     entities enabled</label>
                     <p class="parsley-errors"></p>
                     <input type="checkbox" id="dashboard_bundle_edit_service_type_general_productionEntitiesEnabled"
-                           name="dashboard_bundle_edit_service_type[general][productionEntitiesEnabled]" value="1"
-                           checked="checked"/></div>
```

Comment:
Was the checked state deliberately removed?

Reply:
No, this represents the way the form is rendered. I tried to change as little as possible in that regard.